src/integrationTest/java/uk/gov/hmcts/darts/noderegistration/controller/NodeRegistrationControllerTest.java

```diff
@@ -57,9 +58,14 @@ void testPostRegisterDevices() throws Exception {
     @Test
     void testInvalidCourtroom() throws Exception {
```

Comment:
What is this test trying to do? Like what is an invalid Courtroom? Can we update the test description to refer to the courtroom not existing? 
Also, are we missing a test to cover the scenario for when an exception is thrown for a Courthouse that doesn't exist?

Reply:
i suppose "missing courthroom" woud be a better description

Reply:
done